### ISSUE TITLE
Add "Tabulation Time" to JSON + CSV Summary

### ISSUE THREAD

**Issue by artoonie** [BrightSpots/rcv#732]
Many jurisdictions update preliminary results regularly. Managing which of them is newest can be a problem -- especially when sharing with the public, the public should have an easy way of knowing when the results were generated.

What are your thoughts on including a "Tabulation Time" in the JSON + CSV output for 1.4.0?

(Note: this was requested by Boulder County to include on RCVis, and RCVis primarily uses the RCTab JSON format.)

**Comment by tarheel** [comment]
Makes sense to me. Maybe a start time and an end time? Or, if there's only one, we should specify which one it is.

**Comment by artoonie** [comment]
Would `tabulationCompletedAt` instead of `tabulationTime` address that?

**Comment by tarheel** [comment]
> Would `tabulationCompletedAt` instead of `tabulationTime` address that?

Sure.

**Comment by yezr** [comment]
Looking around for where time is currently displayed I see it **only** in the filename of the output files. It is not included in the content of either summary file or the audit log. 

Where are we thinking this would live? `Tabulation Started` and `Tabulation Completed` in the Contest Information at the top of the `summary.csv`? The same keys in the `config` section of the `summary.json`?

**Comment by artoonie** [comment]
Yes, that's where I was thinking in both the CSV and JSON
